sway-lib-core/src/primitive_conversions.sw

```diff
@@ -647,30 +635,60 @@ impl b256 {
     /// }
     /// ```
     pub fn to_le_bytes(self) -> [u8; 32] {
-        let (a, b, c, d): (u64, u64, u64, u64) = asm(r1: self) {r1: (u64, u64, u64, u64)};
+        let (a, b, c, d): (u64, u64, u64, u64) = asm(r1: self) { r1: (u64, u64, u64, u64) };
         let a = a.to_le_bytes();
         let b = b.to_le_bytes();
         let c = c.to_le_bytes();
         let d = d.to_le_bytes();
 
-        let (a,b,c,d) = (d,c,b,a);
-
-        let output = [a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7],
-                      b[0], b[1], b[2], b[3], b[4], b[5], b[6], b[7],
-                      c[0], c[1], c[2], c[3], c[4], c[5], c[6], c[7],
-                      d[0], d[1], d[2], d[3], d[4], d[5], d[6], d[7]];
+        let (a, b, c, d) = (d, c, b, a);
+
+        let output = [
+            a[0],
+            a[1],
+            a[2],
+            a[3],
+            a[4],
+            a[5],
+            a[6],
+            a[7],
+            b[0],
+            b[1],
+            b[2],
+            b[3],
+            b[4],
+            b[5],
+            b[6],
+            b[7],
+            c[0],
+            c[1],
+            c[2],
+            c[3],
+            c[4],
+            c[5],
+            c[6],
+            c[7],
+            d[0],
+            d[1],
+            d[2],
+            d[3],
+            d[4],
+            d[5],
+            d[6],
+            d[7],
+        ];
```

Comment:
This is a little unsightly. rustfmt would inline them like this:

```rust
    let output = [
        a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7], b[0], b[1], b[2], b[3], b[4], b[5], b[6],
        b[7], c[0], c[1], c[2], c[3], c[4], c[5], c[6], c[7], d[0], d[1], d[2], d[3], d[4], d[5],
        d[6], d[7],
    ];
```

Perhaps we should open an issue.

Reply:
there may already be one IIRC